core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -112,4 +118,31 @@ public RequestHeadersBuilder authority(String authority) {
         setters().authority(authority);
         return this;
     }
+
+    @Override
+    public List<LanguageRange> acceptLanguages() {
+        final HttpHeadersBase getters = getters();
+        if (getters != null) {
+            return getters.acceptLanguages();
+        }
+        return HttpHeadersBase.ACCEPT_LANGUAGE_WILDCARD;
```

Comment:
I changed it according to your suggestion/question to return null on parse error, so while not it's possible to differ from `accept-language: *' and lack of `accept-language` header, it's not impossible to differ between no _valid_ header and no header at all. Tbh, I think that's ok, as I can't see any real value to the server side to differ between the two cases. If there is a case, it could be solved by checking for the existence of the header.